apps/bfd-server/bfd-server-war/src/test/java/gov/cms/bfd/server/war/stu3/providers/ExtraParamsInterceptor.java

```diff
@@ -10,25 +10,16 @@
 /** A HAPI {@link IClientInterceptor} that allows us to add HTTP headers to our requests. */
 public class ExtraParamsInterceptor implements IClientInterceptor {
   private RequestHeaders requestHeader;
-  // private IHttpRequest theRequest;
```

Comment:
@JFU-GIT - Looks like a recent merge removed your comments. These should go back. Apologies.